content/advent-2018/go-in-the-browser.md

```diff
@@ -0,0 +1,175 @@
++++
+author = ["Agniva De Sarker"]
+title = "Go and WebAssembly: running Go programs in your browser"
+linktitle = "Go in the browser"
+date = 2018-12-07T00:00:00Z
+series = ["Advent 2018"]
++++
+
+For a long time, Javascript was the lingua franca amongst web developers. If you wanted to write a stable, mature webapp, writing in javascript was pretty much the only way to go.
+
+WebAssembly is going to that change soon. Using WebAssembly (also called wasm), one can write webapps in _any_ language now. In this post, we will see how to write Go programs and run them in the browser using wasm.
```

Comment:
Corrected. I prefer to abbreviate to wasm instead of Wasm. As I have used wasm consistently throughout the post. I believe there is no rigid rule on this. It is a matter of consistency.